snuba_sdk/legacy.py

```diff
@@ -66,14 +117,81 @@ def parse_exp(value: Any) -> Any:
     return Function(value[0], children, alias)
 
 
+def parse_automatic_condition(
```

Comment:
What about calling this `parse_extension_condition` ? That would make it pretty clear what this is for.